hebiros_advanced_examples/launch/mapping_navigation/amcl.launch

```diff
@@ -0,0 +1,50 @@
+<launch>
+  <!-- Arguments -->
+  <arg name="scan_topic"     default="scan"/>
```

Comment:
This is where the `scan` vs. `scan_filtered` topic is set, right?  We should mention this in the README, or at least tell how to change it with arguments.